### PR TITLE
Add a contributing template - fixes #149

### LINKED ISSUE #149: Add a pull request template file
* Let people know about `#trivial` and CHANGELOG
* Remind people that empty PR descriptions will fail the build
* Note re CoC, and that if they are new they will get an org invite

### PR DESCRIPTION
**Pull request by orta**
Based the issue template on CocoaPods' then went free form on the PR one, mainly to help people not feel foolish about the CHANGELOG stuff

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+`lib/*` - if you're fixing unreleased code, or doing a simple typo change, you can
+include `#trivial` in the title or the body of the PR and this is skipped.
+
+We also request that you fill in the body of your PR, a title should be tweet length, but
+ideally you can explain the PRs reasoning in the body. We look that it's longer than 5 chars.
+
+Other than that, a lot of the other Danger rules are trickier to trigger so we can address
+them as they come up!
+
+THANKS FOR HELPING OUT :D
```
Comment (k0nserv) [inline .github/pull_request_template.md]: Great, needs more emojis though? 👌

**Comment by orta** [comment]
agreed - given it more of a @KrauseFx flair

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+///////////////////⚡///////////////////
+
+# 🚫 AWESOME A PR!
+
+- You can just delete all of this and start your PR text anytime -
+
+Hello there, just a quick pre-warning of some of the Danger rules we on Danger PRs.
```
Comment (marcelofabri) [inline .github/pull_request_template.md]: missing a verb after "Danger rules we"?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+We request that every code change to Danger include a CHANGELOG entry, this is 
+so that:
+
+1. Orta doesn't get all the credit for other people's work
+2. People know what changes are between versions
+
+Danger will look for a modification to the `CHANGELOG.md` when there are changes including
```
Comment (marcelofabri) [inline .github/pull_request_template.md]: maybe link to CHANGELOG.md?

Comment (orta) [inline .github/pull_request_template.md]: I thought about it, but people see these as un-rendered markdown, so a link doesn't add much here

Comment (marcelofabri) [inline .github/pull_request_template.md]: forgot about that, agreed 👍

**Comment by KrauseFx** [comment]
Yes for emojis, thanks @k0nserv

**Pull request merged**